wordpress-mobile/WordPress-iOS
# Simplify the login validation code.

Removes obsolete or unneeded code to check the credentials of a self-hosted site.

How to test:

 * Add a self-hosted site to the app
 * Go to site-> settings
 * Change the password to valid and invalid values and see if the error messages make sense. 

Needs review: @koke
